src/applications/search/containers/SearchApp.jsx

```diff
@@ -480,6 +482,22 @@ class SearchApp extends React.Component {
       };
     }
 
+    if (searchGovMaintenance) {
+      return (
+        <div className="columns vads-u-margin-bottom--4">
+          <va-banner
+            data-label="Error banner"
+            headline="Search.gov Maintenance"
```

Comment:
I think Dave doesn't want us to mention Search.gov. Maybe just 'Search Maintenance'?

Reply:
Good call. Made this update.